image/templates/helm/stackrox-secured-cluster/templates/collector.yaml.htpl

```diff
@@ -100,6 +104,18 @@ spec:
         - mountPath: /run/secrets/stackrox.io/certs/
           name: certs
           readOnly: true
+        [<- if .FeatureFlags.ROX_COLLECTOR_RUNTIME_CONFIG >]
+        {{- if and ._rox.collector.containerd.enabled (not ._rox.env.openshift) }}
```

Comment:
Just FYI, I cannot comment on the actual logic here.

Reply:
I've changed this in favor of setting `containerd.enabled` and `crio.enabled` in the defaults files. The logic should be to default to using containerd on k8s clusters and crio on openshift.